### ISSUE TITLE
crash nextcloud dev android after start aplication

### ISSUE THREAD

**Issue by matuszajac** [nextcloud/android#8105]
### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
************ CAUSE OF ERROR ************

java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.util.ArrayList.get(int)' on a null object reference
	at com.owncloud.android.lib.common.operations.RemoteOperationResult.getSingleData(RemoteOperationResult.java:527)
	at com.owncloud.android.operations.RefreshFolderOperation.updatePredefinedStatus(RefreshFolderOperation.java:339)
	at com.owncloud.android.operations.RefreshFolderOperation.updateCapabilities(RefreshFolderOperation.java:304)
	at com.owncloud.android.operations.RefreshFolderOperation.updateOCVersion(RefreshFolderOperation.java:276)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:229)
	at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:359)
	at java.lang.Thread.run(Thread.java:919)

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20210304
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: motorola
Device: sofiar
Model: moto g(8) power
Id: QPES30.79-124-9
Product: sofiar_reteu

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 97554


```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
This will be fixed via #8106

**Comment by joshtrichards** [comment]
Fixed in #8106
